### PR TITLE
switch to new implementation of start-cbinaries.sh [run-systemtest]

### PR DESCRIPTION
**Pull request by arnej27959**
@baldersheim please review
I think we can give this a try on factory.
@aressem @hmusum FYI

### CODE REVIEW

**Review by baldersheim — APPROVED** [review]
Let us run with basic system test to ensure basics works

**Comment by baldersheim** [comment]
Not quite right yet.
 14:51:55 [2022-09-30 12:36:43.342] WARNING searchnode       stderr	/bin/sh: /opt/vespa/sbin/vespa-proton: No such file or directory
14:51:55 [2022-09-30 12:36:43.342] WARNING distributor      stderr	/bin/sh: /opt/vespa/sbin/vespa-distributord: No such file or directory

**Review by baldersheim — CHANGES_REQUESTED** [review]
See runtime errors

File: storageserver/src/apps/storaged/CMakeLists.txt [hunk 1]
```diff
@@ -9,4 +9,4 @@ vespa_add_executable(storageserver_storaged_app
     storageserver_storageapp
 )
 
-install(CODE "execute_process(COMMAND ln -snf vespa-storaged-bin \$ENV{DESTDIR}/\${CMAKE_INSTALL_PREFIX}/sbin/vespa-distributord-bin)")
+install_absolute_symlink(vespa-storaged-bin sbin/vespa-distributord-bin)
```
Comment (toregge) [inline storageserver/src/apps/storaged/CMakeLists.txt:12]: The first argument to` install_absolute_symlink` needs to start with `sbin/`

**Review by baldersheim — APPROVED** [review]

**Pull request merged**